Automattic/woocommerce-payments
# Move E2E user account management to setup script

Fixes #3451 

#### Changes proposed in this Pull Request

At the moment all E2E tests are make use of already created user accounts and doesn't require creating new ones. To optimize the tests, we can move user account creation and guest account removal (if any) to the E2E setup script rather than running before every test suite.

#### Testing instructions

* Confirm that user account is created & guest account is removed (if any) while running the E2E setup locally.
* On GH actions, trigger E2E test against this branch and confirm that the user account is created & guest account is removed during setup.

-------------------


**Post merge**
